imagersite/imagerprofile/app.py

```diff
@@ -0,0 +1,9 @@
+from django import AppConfig
+
+
+class ImagerProfileConfig(AppConfig):
+    name = 'imagerprofile'
+    verbose_name = 'Imager Profile'
+
+    def ready(self):
+        import handlers
```

Comment:
Why is this file still here?  It's identical to the `apps.py` file below, and that's the one being used as the `default_app_config` in `__init__.py` above.  Remove useless files to save your sanity.